src/xrpld/app/tx/detail/MPTokenAuthorize.cpp

```diff
@@ -83,6 +83,18 @@ MPTokenAuthorize::preclaim(PreclaimContext const& ctx)
                 return tecHAS_OBLIGATIONS;
             }
 
+            if (ctx.view.rules().enabled(featureTokenEscrow) &&
+                (*sleMpt).isFieldPresent(sfEscrowedAmount) &&
+                (*sleMpt)[~sfEscrowedAmount] != 0)
```

Comment:
A similar check should be added in `MPTokenIssuanceDestroy` tx as well. We shouldn't allow the deletion of an `MPTokenIssuance` if `sfEscrowedAmount` is non-zero